portfolio/src/main/webapp/script.js

```diff
@@ -22,12 +22,26 @@ function openModal(num) {
 
 function fetchFunction() {
     fetch('/data').then(response => response.json()).then((data) => {
-        commentsArray = data;
+        commentsArray = data.comments;
         start = 0;
         if (commentsArray.length === 0) {
             document.getElementById('commentDependent').setAttribute('class', 'hide');
         }
         assginCommentsToUI();
+
+        if (!data.user) {
+            document.getElementById('commentForm').setAttribute('class', 'hide');
+            document.getElementById('loginBtn').setAttribute('href', data.url);
+            console.log("no user");
```

Comment:
If this is for debugging purpose, let's remove it before merging

Reply:
Yes, I forgot to delete them, thanks